docs/langref.md

```diff
@@ -813,6 +790,50 @@ import { Logic as MathLogic } from "math.slint";
 export { MathLogic } // known as "MathLogic" when using native APIs to access globals
 ```
 
+<details>
+<summary>Usage from Rust</summary>
+
```

Comment:
A link to the ``[`slint::Global`]`` would be great as it has some example (and could get more) https://docs.rs/slint/latest/slint/trait.Global.html

Reply:
I can try to do that, but there are two problems:

(1) It will be a broken link in the C++ docs. That can be worked around with the hiding, but that's not approved yet.
(2) It will be a broken link when looking at the reference in GitHub.

If you don't mind that then I can go ahead and work more on this to add a link to the Rust global trait and the C++ global docs. Let me know :)